app/pages/organizations/organization-page.jsx

```diff
@@ -183,11 +193,39 @@ class OrganizationPage extends React.Component {
             </div>
             <div className="organization-researcher-words">
               <h4 className="organization-details__heading"><Translate content="organization.home.links" /></h4>
-              <ul>
-                <li className="organization-details__link">Blog</li>
-                <li className="organization-details__link"><i className="fa fa-facebook fa-fw" />- @orgFacebook</li>
-                <li className="organization-details__link"><i className="fa fa-twitter fa-fw" />- @orgTwitter</li>
-              </ul>
+              <div className="organization-details__links">
+                {!!rearrangedLinks.length && rearrangedLinks.map((link, i) => {
+                  if (link.path) {
+                    let iconForLabel;
+                    if (Object.keys(SocialIcons).indexOf(link.site) >= 0) {
+                      iconForLabel = SocialIcons[link.site];
+                    } else {
+                      iconForLabel = 'globe';
+                    }
+                    return (
+                      <a
+                        key={i}
```

Comment:
For the key I think I should use `link.key` for external and `link.path` for social links instead of the array index.